code/game/objects/structures/morgue.dm

```diff
@@ -155,7 +155,12 @@
 	return
 
 /obj/structure/morgue/Destroy()
-	QDEL_NULL(connected)
+	if(!connected)
+		var/turf/T = src.loc
+		for(var/atom/movable/A as mob|obj in src)
+			A.forceMove(T)
```

Comment:
```suggestion
		var/turf/T = loc
		for(var/atom/movable/A in src)
			A.forceMove(T)
```

Reply:
Mmm. I trust your suggestions and I'm going to test them before committing them.